sound/soc/codecs/cs42l42.c

```diff
@@ -646,6 +646,7 @@ static const struct cs42l42_pll_params pll_ratio_table[] = {
 	{ 3072000,  1, 0x00, 0x3E, 0x800000, 0x03, 0x10, 12000000, 125, 1},
 	{ 4000000,  1, 0x00, 0x30, 0x800000, 0x03, 0x10, 12000000,  96, 1},
 	{ 4096000,  1, 0x00, 0x2E, 0xE00000, 0x03, 0x10, 12000000,  94, 1},
+	{ 4800000,  1, 0x01, 0x50, 0x000000, 0x03, 0x10, 12000000, 128, 2},
```

Comment:
we need to have support for 9.6 MHz as well. Also we need support for the 12.288 family. This needs to be handled with Cirrus Logic, adding @rfvirgil for comments.

Reply:
Please suggest if 9.6 MHz needs to be introduced as part of this PR